src/wallet/wallet.cpp

```diff
@@ -2583,6 +2583,7 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
                          int& nChangePosInOut, std::string& strFailReason, const CCoinControl& coin_control, bool sign)
 {
     CAmount nValue = 0;
+    nFeeRet = 0;
```

Comment:
is this valuable by itself?

Reply:
Probably. I had a garbage value being returned and it ended up telling me "you need at least 4587368374.13587638 bitcoin in fees!" here:

https://github.com/bitcoin/bitcoin/blob/3f826598a42dcc707b58224e94c394e30a42ceee/src/wallet/rpcwallet.cpp#L353-L357

Reply:
PR-ified as #18274.